src/CLI/clitree/cli-xml/interface.xml

```diff
@@ -281,8 +281,19 @@ limitations under the License.
           </PARAM>
        <ACTION builtin="clish_nop"></ACTION>
        </COMMAND>
+       <COMMAND name="sflow" help="enable sflow">
+            <PARAM name="sampling-rate" help="sFlow sampling rate" ptype="SUBCOMMAND" mode="subcommand" optional="true">
+                <PARAM name="rate" help="sampling rate" ptype="UINT"/>
+            </PARAM>
+            <PARAM name="admin-status" help="sFlow status for interface" ptype="SUBCOMMAND" mode="subcommand" optional="true">
```

Comment:
In your HLD, this parameter was called `enable` rather than `admin-status`.

If both of the PARAMs under here are `optional="true"` then will this result in a user being able to configure just `sflow` by itself? I don't think we should allow that.

Reply:
BTW, I think `enable` is a better keyword than `admin-status`, since "admin status" is typically used to describe the enable/disable of an interface.

Reply:
will change.